sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -272,15 +258,16 @@ def __next__(self):
 
     next = __next__  # for python2.7
 
-    def _iter_next(self):
+    def _iter_next(self, wait_time=None):
         try:
             self._receive_context.set()
             self._open()
-            # TODO: Add in Recieve Message Iterator
-            # if not self._message_iter:
-            #     self._message_iter = self._handler.receive_messages_iter()
-            uamqp_message = next(self._message_iter)
-            message = self._build_message(uamqp_message)
+            if not self._message_iter:
+                self._message_iter = self._handler.receive_messages_iter(timeout=wait_time)
+            elif wait_time:
+                self._message_iter = self._handler.receive_messages_iter(timeout=wait_time)
```

Comment:
These seem to be doing the same thing - couldn't they be merged into the same if statement?